src/modules/preview-site/components/create-preview-button.tsx

```diff
@@ -24,10 +29,23 @@ export function CreatePreviewButton( { onClick, selectedSite }: CreatePreviewBut
 	const { isOverLimit } = useSiteSize( selectedSite.id );
 	const isOffline = useOffline();
 	const errorMessages = useArchiveErrorMessages();
+	const [ wpVersion ] = useGetWpVersion( selectedSite );
+	const wpVersions = useRootSelector( wordpressVersionsSelectors.selectWordPressVersions );
 
 	const isCurrentSiteArchiving = archivingSiteId === selectedSite.id;
 	const isOtherSiteArchiving = isAnySiteArchiving && ! isCurrentSiteArchiving;
 
+	// Jurrasic Ninja will use the default version for PHP and latest or above for WordPress.
+	const latestWpVersion = wpVersions.find( ( version ) => version.isBeta === false )?.value;
+	const isPhpVersionDefault = selectedSite.phpVersion === DEFAULT_PHP_VERSION;
+	const coercedWpVersion = semver.coerce( wpVersion );
+	const coercedLatestWpVersion = semver.coerce( latestWpVersion );
+	const isWpVersionBelowDefault =
```

Comment:
Could we extract the logic around `isPhpVersionDefault` and `isWpVersionBelowDefault` to a separate function, e.g., `shouldShowTooltip`, and add some tests covering possible cases?

Reply:
Yeah, good points 👍

Also, in this context, let's consider these two my comments [1](https://github.com/Automattic/studio/pull/1059#discussion_r1993671061) and [2](https://github.com/Automattic/studio/pull/1059#discussion_r1993674275).